pkg/local/sorted_queue.go

```diff
@@ -1,28 +1,33 @@
 package local
 
-// A SortedQueue implements heap.Interface and holds strings.
-type SortedQueue []string
+// A StringHeap is a min-heap of strings
+type StringHeap []string
```

Comment:
I don't understand why any method of this object ever takes an `any` argument: it holds only `string`s, which is fine.  Why do we want to cast?

Reply:
We have to maintain the interface which returns `any`, otherwise we could not use the heap functionality
See: https://pkg.go.dev/container/heap#Interface